### DISCUSSION TITLE
Panels within panels QOL

### DISCUSSION THREAD

**Discussion by No2515** [mbnuqw/sidebery#1535] [Ideas]
Just for more organization. If I'm shopping, I'd like to only focus on shopping. So I'd have separate panels for Amazon, ebay, etc. Preferably, using icons at the bottom to switch stores. 

If I need to return to work with my other panels, scroll on the nav bar up top.

**Comment by zelch** [comment]
There are some things that you can do, with various levels of how well it works. (Doing a google search in one tab, and then trying to open Amazon/whatever tabs out of that tends to work poorly with this approach.)

The biggest is that you can setup panels, with rules on what sites will automatically open in a given panel.

That way all Amazon tabs will open in your Amazon panel, etc.

This doesn't give you the bottom panel controls, sadly, but it does give you store specific panels.

**Comment by No2515** [comment]
That's true, but if you're shopping multiple items from the same store, the panel fills quickly. 

Furthermore, shopping is just one example. If I'm researching subjects from various topics over a multiple sites, the folder count adds up. 

As of now, I'm averaging 5 folders per panel; 10 panels in my navigation bar. If I could reduce the nav bar to 3 or 4 and switch to other projects with a click of a button, it would clean things up immensely
